Package.swift

```diff
@@ -11,4 +12,24 @@
 
 import PackageDescription
 
-let package = Package(name: "DeviceKit")
+let package = Package(
+    name: "DeviceKit",
+    products: [
+        // Products define the executables and libraries produced by a package, and make them visible to other packages.
+        .library(
+            name: "DeviceKit",
+            targets: ["DeviceKit"])
+    ],
+    targets: [
+        // Targets are the basic building blocks of a package. A target can define a module or a test suite.
+        // Targets can depend on other targets in this package, and on products in packages which this package depends on.
+        .target(
+            name: "DeviceKit",
```

Comment:
Could you specify the `Source` path here instead of it using the default `Sources` path. Then we can remove the shortcut.
https://github.com/apple/swift-package-manager/blob/master/Documentation/PackageDescription.md#methods-5

Reply:
Yes, I've done another commit to fix this issue. 
Yesterday I overlooked the documentation, I'm sorry.